File: homafo_frontend/src/models/dto/AforoPreLiquidacionResponse.ts
import DetalleAforoDTO from "./DetalleAforoDTO";
import GeneradorDTO from "./GeneradorDTO";
import MaestroVisitasResource from "./MaestroVisitasResource";

export default interface AforoPreLiquidacionResponse {
    aforo:number;
	minimoVisitas:number;
	visitasTramitadas:number;
	totalVisitasConsolidado:number;
	volumenMedio:number;
	generadores: GeneradorDTO[];
	maestroVisitas:MaestroVisitasResource;
	detalleAforo: DetalleAforoDTO[]; //detalles
	valido:boolean;
	mensaje:string;
	tipoAforo:number;
	genFactorEquivalencia:number;

}

File: homafo_backend/src/main/java/com/bioagricola/aforos/entity/dto/AforoPreLiquidacionResponse.java
package com.bioagricola.aforos.entity.dto;

import java.util.List;

import lombok.Data;

@Data
public class AforoPreLiquidacionResponse {


	private Long aforo;
	private Long minimoVisitas;
	private Long visitasTramitadas;
	private Double totalVisitasConsolidado;
	private Double volumenMedio;
	private Double pesoMultiusuario;
	private Boolean valido;
	private String mensaje;
	private List<GenGeneradorResource> generadores;
	private List<DetalleAforoInfoDTO> detalleAforo;
	private MaestroVisitasResource maestroVisitas;
	private Long tipoAforo;
}
